tests/simulator/test_vtk_diagnostics.py

```
@@ -0,0 +1,185 @@
+    "final_time": 0.003,
+    "boundary_types": "periodic",
+    "cells": 40,
+    "dl": 0.3,
+    "diag_options": {
+        "format": "phareh5",
+        "options": {"dir": out, "mode": "overwrite"},
+    },
+}
+
+
+def permute(dic={}):
```

Ruff (0.14.3)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)